lib/yeti_logger/custom_formatter.rb

```diff
@@ -0,0 +1,33 @@
+class YetiLogger::CustomFormatter
+  # @param tags [Hash] - maps names of tags to procs that return their value
+  def initialize(tags = {})
+    super()
+    @tags = tags
+  end
+
+  # @param tags [Hash] - maps names of tags to procs that return their value
+  def add_tags(tags)
+    @tags.merge!(tags)
+  end
+
+  # @param severity [String]
+  # @param time [Time] unused
+  # @param progname [String] unused
+  # @param msg [String]
+  def call(severity, time, progname, msg)
+    timestamp = "#{Time.now.iso8601(3)}"
```

Comment:
Should we use UTC? 
```suggestion
    timestamp = Time.now.utc.iso8601(3)
```

Reply:
Great catch. UTC would align with the format currently used by application logs

![Screenshot 2024-03-01 at 4 09 17 PM](https://github.com/vendasta/yeti_logger/assets/83599192/93e5907d-8f90-4067-8eed-53950b901393)